apps/playground-lit/src/app/app.element.ts

```diff
@@ -1,17 +1,18 @@
 import { CSSResult, html, LitElement, TemplateResult } from 'lit';
 import { customElement } from 'lit/decorators.js';
 import { vlElementsStyle } from '@domg-wc/elements';
-import { VlAccordionComponent, VlCascaderComponent, VlInfoTile } from '@domg-wc/components';
+import { VlAccordionComponent, VlCascaderComponent, VlInfoTile, VlPagerComponent } from '@domg-wc/components';
 import { registerWebComponents } from '@domg-wc/common-utilities';
 import appElementStyle from './app.element.css';
 import { cascaderItemTemplates } from './vl-cascader.templates';
 import { getItemList } from './vl-cascader.utils';
 import { nodeData } from './vl-cascader.data';
+import { VlHeader } from '@domg-wc/sections';
 
 @customElement('app-element')
 export class AppElement extends LitElement {
     static {
-        registerWebComponents([VlCascaderComponent, VlInfoTile, VlAccordionComponent]);
+        registerWebComponents([VlCascaderComponent, VlInfoTile, VlAccordionComponent, VlHeader, VlPagerComponent]);
```

Comment:
normaalgezien gaan we de playgrounds niet aanpassen, behalve voor nieuwe componenten

wat ik echter wel soms doe, is dat ik soms een aparte commit maak; waar ik het component demonstreer in de playground zodat de reviewers dit makkelijk kunnen zien in de andere environments (maar na approval drop ik die commit dan)

Reply:
opgelost - verwijdeerd